examples/sandbox/index.ts

```diff
@@ -2480,36 +2474,32 @@ $osmosisSwap.on("click", async (e) => {
         * Bech32: false
 
 */
+interface EthOperationConfig {
+  handler: (paths: any) => Promise<any>;
+  useEIP1559?: boolean;
+}
 
 let ethEIP1559Selected = false;
 
-const ethTx = {
-  addressNList: core.bip32ToAddressNList("m/44'/60'/0'/0/0"),
-  nonce: "0x01",
-  gasPrice: "0x1dcd65000",
-  gasLimit: "0x5622",
-  value: "0x2c68af0bb14000",
-  to: "0x12eC06288EDD7Ae2CC41A843fE089237fC7354F0",
-  chainId: 1,
-  data: "",
-};
-
-const ethTx1559 = {
-  addressNList: core.bip32ToAddressNList("m/44'/60'/0'/0/0"),
-  nonce: "0x0",
-  gasLimit: "0x5ac3",
-  maxFeePerGas: "0x16854be509",
-  maxPriorityFeePerGas: "0x540ae480",
-  value: "0x1550f7dca70000", // 0.006 eth
-  to: "0xfc0cc6e85dff3d75e3985e0cb83b090cfd498dd1",
-  chainId: 1,
-  data: "",
-};
+const ethButtons = [
+  "ethAddr",
+  "ethTx",
+  "ethSign",
+  "ethSend",
+  "ethVerify",
+  "ethResults",
+  "ethEIP1559",
+  "ethSignTypedData",
+  "ethSignTypedDataAlternate",
+].reduce((acc, id) => {
+  acc[id] = $(`#${id}`);
+  return acc;
+}, {} as Record<string, JQuery<HTMLElement>>);
```

Comment:
This is a low-risk refactor that ultimately helped me find and fix the bug faster using the powers of TypeScript while also eliminating repetitive code